docs/source/deployment/heroku.rst

```diff
@@ -105,6 +105,10 @@ BentoService and available for sending test request:
 Build and deploy to Heroku
 ==========================
 
+.. note::
+  Apps deployed on Heroku must listen to a specific port number which is
+  specified by the $PORT environment variable in the dyno. Bentoml will
```

Comment:
is it a typo?  I guess you mean `...in the demo`?

Reply:
no dyno is what heroku calls their containers, so I thought I'd use the same term here.